### ISSUE TITLE
any chance of adding an analog clock / calendar popup AKA Windows 7 ?

### ISSUE THREAD

**Issue by rancorx2** [White-Tiger/T-Clock#145]
good program but i'd like it if the calendar popup looked like the windows 7 one and had analog clock on it, would this be a possible addition? would be perfect if this could be done, don't like the massive windows 10 calendar ans digi clock.

**Comment by White-Tiger** [comment]
Well... T-Clock did that, that's why it had the "Restore previous calendar" option...
But newer Windows 10 versions removed that part entirely. So there's no way to get it back _(well, technically there is, one could use the Windows 8 dll that opened the Calendar pop out. Though I'm not sure if T-Clock would be legally allowed to ship "internal" Windows dll files)_

**Comment by rancorx2** [comment]
which dll did the calendar pop out?

**Comment by rancorx2** [comment]
the only dll i could possible connect it with is shell32.dll, was that wrong? if so i'd love to know which dll makes the clock pop-up???

**Comment by White-Tiger** [comment]
I don't know which one exactly.. might have been one with a .cpl extension... Someone posted stuff about it, but I forgot who and where

**Comment by Steepleman** [comment]
Does that mean if we take the old timedate.cpl and replace the current (new) one, we can get the old clock/calendar back?

**Comment by ghost** [comment]
doesn't work, the cpl is for time / date setting in control panel, the tray clock is part of the shell, if i'm correct.

**Comment by Steepleman** [comment]
Yes, that's true. However, it seems the old Win32 calendar is still included, but just inaccessible... or hard to access, like the old Personalisation windows which can be accessed through explorer shell.

**Comment by White-Tiger** [comment]
`timedate.cpl` is indeed the Calendar, Tooltip and Analog clock. You could use them with COM calls.

**Comment by Steepleman** [comment]
Is there any chance of T-Clock readding it then?

**Comment by White-Tiger** [comment]
reading? reading what?

**Comment by Steepleman** [comment]
Re-adding the tray clock.
